### ISSUE TITLE
Fails to install via zef because of test failures

### ISSUE THREAD

**Issue by ryn1x** [ab5tract/Terminal-Print#60]
```
===> Testing: Terminal::Print:ver<0.94>
t/00-basics.t .. ok
t/10-grid.t .... All 1 subtests passed 
t/11-golf.t .... All 1 subtests passed 
All tests successful.

Test Summary Report
-------------------
t/10-grid.t  (Wstat: 0 Tests: 1 Failed: 0)
  Parse errors: Subtest 1 expected 3 but contains 0 tests
t/11-golf.t  (Wstat: 0 Tests: 1 Failed: 0)
  Parse errors: Subtest 1 expected 2 but contains 0 tests
Files=3, Tests=6,  15 wallclock secs
Result: FAILED
===> Testing [FAIL]: Terminal::Print:ver<0.94>
Aborting due to test failure: Terminal::Print:ver<0.94> (use --force-test to override)
```

**Comment by lizmat** [comment]
@ryn1x: On which OS is this?  I also have problems installing `Terminal::Print` on MacOS, but that's related to "xterm-color" not being a known terminal type.  Adding that to the list of supported types, creates errors such as:

    Tried to use an undefined capability 'civis' of the terminal type 'xterm-color'

So I'm not sure this is the same issue.

**Comment by ryn1x** [comment]
@lizmat 

I use: openSuse Tumbleweed.

And for Perl6: This is Rakudo version 2018.11-4-g38bc682b0 built on MoarVM version 2018.11
implementing Perl 6.d.

#61 fixed the issue for me. I don't know why this would happen only on some people's systems though... Has the Test module changed with newer versions of Perl6?

**Comment by lizmat** [comment]
@ryn1x Nah, the issue I see is about actually failing tests, rather than test-counting going wrong.  So I guess I will need to create a separate issue.

**Comment by rabbiveesh** [comment]
I'm having this problem also on Ubuntu with version 2018.04.1 built on MoarVM version 2018.04.1
implementing Perl 6.c.  What's odd is that when i `zef look` into the directory, and `make test`, all of them pass...
